XpringKit/PayID/PayIDClient.swift

```diff
@@ -29,20 +33,53 @@ public class PayIDClient {
   ///   - ach
   ///
   //  TODO: Link a canonical list at payid.org when available.
-  public init(network: String) {
+  public init(network: String, completionQueue: DispatchQueue = DispatchQueue.main) {
```

Comment:
is this needed?

Reply:
Agreed. This can be removed. The network and callback queues are already in place. Plus, AFAIK, this is a breaking change as far as bitcode is concerned since it's a public API.

Reply:
Apologies. I was heads down on the PayID release last week. Sorry to be sloppy and send out incomplete PRs. 

This is resolved.